### ISSUE TITLE
Circular dependency between dev-tools and themes.

### ISSUE THREAD

**Issue by moniika** [RaspberryPiFoundation/blockly-samples#785]
**Category**

Plugins

**Component**

dev-tools and most of the theme plugins

**Describe the bug**

Circular dependency between dev-tools and themes, causing a warning and sometimes an error when testing github pages locally.

**To Reproduce**

1. Host gh-pages locally `npm run test:ghpages:beta`
2. Observe warning in console about dependencies:

```
lerna WARN ECYCLE Dependency cycles detected, you should fix these!
lerna WARN ECYCLE @blockly/dev-tools -> @blockly/theme-dark -> @blockly/dev-tools
lerna WARN ECYCLE @blockly/theme-deuteranopia -> (nested cycle: @blockly/dev-tools -> @blockly/theme-dark -> @blockly/dev-tools) -> @blockly/theme-deuteranopia
lerna WARN ECYCLE @blockly/theme-highcontrast -> (nested cycle: @blockly/theme-deuteranopia -> (nested cycle: @blockly/dev-tools -> @blockly/theme-dark -> @blockly/dev-tools) -> @blockly/theme-deuteranopia) -> @blockly/theme-highcontrast
lerna WARN ECYCLE @blockly/theme-tritanopia -> (nested cycle: @blockly/theme-highcontrast -> (nested cycle: @blockly/theme-deuteranopia -> (nested cycle: @blockly/dev-tools -> @blockly/theme-dark -> @blockly/dev-tools) -> @blockly/theme-deuteranopia) -> @blockly/theme-highcontrast) -> @blockly/theme-tritanopia
lerna WARN ECYCLE (nested cycle: @blockly/theme-tritanopia -> (nested cycle: @blockly/theme-highcontrast -> (nested cycle: @blockly/theme-deuteranopia -> (nested cycle: @blockly/dev-tools -> @blockly/theme-dark -> @blockly/dev-tools) -> @blockly/theme-deuteranopia) -> @blockly/theme-highcontrast) -> @blockly/theme-tritanopia) -> (nested cycle: @blockly/theme-tritanopia -> (nested cycle: @blockly/theme-highcontrast -> (nested cycle: @blockly/theme-deuteranopia -> (nested cycle: @blockly/dev-tools -> @blockly/theme-dark -> @blockly/dev-tools) -> @blockly/theme-deuteranopia) -> @blockly/theme-highcontrast) -> @blockly/theme-tritanopia)
```

**Expected behavior**

No circular dependencies.

**Additional context**

Similar issue to https://github.com/google/blockly-samples/issues/523

**Comment by moniika** [comment]
This seems to cause issues with running ghpages locally.
Sometimes when I run `npm run test:ghpages:beta`, it fails with the error:
```
removed 10 packages, updated 4 packages and audited 124 packages in 13.149s
npm ERR! code ENOENT | extract:ws: sill extract ws@^7.0.0 extracted to /usr/local/google/home/kozbial/github/b
npm ERR! syscall access
npm ERR! path /usr/local/google/home/kozbial/github/blockly/samples-moniika/plugins/theme-dark/node_modules/@blockly/dev-tools/node_modules/@blockly/theme-dark
npm ERR! errno -2
npm ERR! enoent ENOENT: no such file or directory, access '/usr/local/google/home/kozbial/github/blockly/samples-moniika/plugins/theme-dark/node_modules/@blockly/dev-tools/node_modules/@blockly/theme-dark'
npm ERR! enoent This is related to npm not being able to find a file.
```

However, running the command again usually works.

**Comment by alschmiedt** [comment]
I think this is going to be the same fix as PR #740. However, themes do have an example page on our gh-pages site. So we will also need to double check that the styling is correct for a regular playground instead of an advanced playground.
